Update metadata-api.md (#3008)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-cloud-apis/metadata-api.md

## Product roadmap

The Metadata API roadmap includes initiatives to improve each of the API's main use cases: data discovery, data quality, and dbt operations

<!--- tabs for discovery, quality, operations --->
<Tabs>

<TabItem value="discovery" label="Discovery">

To improve discovery experiences, we’ll make it easier for API users to access the latest production state of a project. This is our primary focus in 2023. 

- Rather than querying for each job or run, retrieve the logical state (definitions), execution results, and applied state (what exists in the database) of each node in the project based on their most recent runs in the production environment.
- dbt v1.6: Use public models to enable [multi-project deployments](https://github.com/dbt-labs/dbt-core/discussions/6725) and access the global lineage
- dbt v1.6: New and revised model and entity endpoints to unlock Semantic Layer use cases
- Pagination for manageable responses and performance when querying long lists of dbt objects

</TabItem>

<TabItem value="quality" label="Quality">

> To improve customers’ data quality, we’re enhancing the ability for data teams to configure dbt and monitor for issues in dbt Cloud and integrated tools. This is our primary focus in 2024. 

Author: ```suggestion
To improve customers’ data quality, we’re enhancing the ability for data teams to configure dbt and monitor for issues in dbt Cloud and integrated tools. This will be more of a focus in 2024.
```

## Product roadmap

The Metadata API roadmap includes initiatives to improve each of the API's main use cases: data discovery, data quality, and dbt operations

<!--- tabs for discovery, quality, operations --->
<Tabs>

<TabItem value="discovery" label="Discovery">

To improve discovery experiences, we’ll make it easier for API users to access the latest production state of a project. This is our primary focus in 2023. 

- Rather than querying for each job or run, retrieve the logical state (definitions), execution results, and applied state (what exists in the database) of each node in the project based on their most recent runs in the production environment.
- dbt v1.6: Use public models to enable [multi-project deployments](https://github.com/dbt-labs/dbt-core/discussions/6725) and access the global lineage
- dbt v1.6: New and revised model and entity endpoints to unlock Semantic Layer use cases
- Pagination for manageable responses and performance when querying long lists of dbt objects

> - Retrieve project information from the dbt logs, such as catalog information, without having to generate documentation.

Reviewer: We can remove this one for now

> - Retrieve project information from the dbt logs, such as catalog information, without having to generate documentation.

Author: ```suggestion

```

</TabItem>

<TabItem value="quality" label="Quality">

> To improve customers’ data quality, we’re enhancing the ability for data teams to configure dbt and monitor for issues in dbt Cloud and integrated tools. This is our primary focus in 2024. 

Reviewer: "This will be more of a focus in 2024" - operations will likely continue into 2024
